src/components/CompareResults/beta/NoResultsFound.tsx

```diff
@@ -16,7 +16,7 @@ function NoResultsFound() {
   }; 
 
   return (
```

Comment:
Are we going to add the `Learn more` button here or in another pr?

Reply:
In another PR it's best as we still need to decide where the user is redirected to after clicking this button.